auxlib/__init__.py

```diff
@@ -1,19 +1,23 @@
 # -*- coding: utf-8 -*-
 """auxiliary library to the python standard library"""
 from __future__ import absolute_import, division, print_function
-import os
 
-from auxlib.__about__ import (
-    __author__, __copyright__, __email__, __license__, __summary__, __title__,
-    __homepage__
-)
+from .packaging import BuildPyCommand, SdistCommand, Tox, get_version
 
 __all__ = [
     "__title__", "__version__", "__author__",
     "__email__", "__license__", "__copyright__",
     "__summary__", "__homepage__",
+    "BuildPyCommand", "SdistCommand", "Tox", "get_version",
 ]
 
-vfile = os.path.join(os.path.dirname(__file__), '.version')
-with open(vfile, 'r') as f:
-    __version__ = f.read().strip()
+__version__ = get_version(__file__, __package__)
+
+__title__ = "auxlib"
+__author__ = 'Kale Franz'
+__email__ = 'kale@franz.io'
+__homepage__ = 'https://github.com/kalefranz/auxlib'
+__license__ = "ISC"
+__copyright__ = "(c) 2015 Kale Franz. All rights reserved."
+__summary__ = gs
+__doc__
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Statement seems to have no effect](https://www.codacy.com/app/kalefranz/auxlib/file/2131390326/issues/source?bid=3138661&fileBranchId=3139130#l23)